ui/src/warp_runner/mod.rs

```diff
@@ -24,7 +29,8 @@ type Storage = Box<dyn Constellation>;
 type Messaging = Box<dyn RayGun>;
 
 pub enum WarpEvent {
-    None,
+    RayGun(Box<RayGunEventKind>),
```

Comment:
Im guessing the events are boxed due to clippy? If so, those can be ignored in this case, though nothing is wrong with having it there if you dont want to ignore the clippy error by using `allow`

Reply:
yes, clippy complained that one variant was 17 bytes while the other was 240 bytes. but having to box everything is annoying.